guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md

```diff
@@ -16,6 +16,8 @@ The main disadvantage of this approach is that Magento applies changes blindly.
 Declarative setup is based on database structure declarations, and is used in projects such as [Doctrine](http://www.doctrine-project.org/). Schema files declare what the database structure should be,
 and Magento determines the differences between the current table structure and what it should be. These differences can be represented with atomic SQL operations.
 
+Magento prioritize declarative schema and executes declarative install schemas before the [data and schema patches]({{ page.baseurl }}/extension-dev-guide/declarative-schema/data-patches.html).
```

Comment:
```suggestion
Magento prioritizes declarative schema and executes the declarative schemas processing before the [data and schema patches]({{ page.baseurl }}/extension-dev-guide/declarative-schema/data-patches.html).
```

Reply:
Could you also review the references (links) you provide in the description? I believe we can use the same set of references for both documents. Currently, you have a reference (link) in the `data-patches.md` that is absent in `db-schema.md` and vice versa: 

[data and schema patches]({{ page.baseurl }}/extension-dev-guide/declarative-schema/data-patches.html)
[db_schema.xml]({{ page.baseurl }}/extension-dev-guide/declarative-schema/db-schema.html)

Thank you!

Reply:
**fixed**

Thank you, @rogyar !